system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -6193,21 +6193,21 @@ class CLdapClient : implements ILdapClient, public CInterface
                         OUarray.appendList(val,"/",true);//create StringArray of OU elements. LDAP returns them in the form "internal.sds/files/dataland_ecl/hpccinternal/roxieuser
 
                         //Find index of first StringArray item NOT in base search string (basednbuf)
-                        int finger = 1;//skip the first element, which are the combined domains. So "dc=internal,dc=sds" will appear here as "internal.sds"
+                        int start = 1;//skip the first element, which are the combined domains. So "dc=internal,dc=sds" will appear here as "internal.sds"
```

Comment:
sorry, but start is a a worse name because you are incrementing it.  (It would be good if it never changed.)  Something like current or currentLevel would be better.

Reply:
Agree